rep-0003.rst

```diff
@@ -299,6 +299,18 @@ Architectures Supported:
 - arm32
 - arm64
 
+Binary Package Availabilty:
```

Comment:
Typo: Should be `Availability`.

But I'm also wondering whether we should use another phrase.  The binary packages will still be "available" after that date, but we won't accept changes to them or build new ones after that date.  Maybe "Binary Package Support"?

Reply:
Availabilty -> Support in 706de1d

Reply:
Also added comments about what support means in cd11c95